pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -204,31 +210,25 @@ type InstanceStatus struct {
 
 // InstanceCondition represents an aspect of an Instance's status.
 type InstanceCondition struct {
-	Type    InstanceConditionType `json:"type"`
-	Status  ConditionStatus       `json:"status"`
-	Reason  string                `json:"reason"`
-	Message string                `json:"message"`
+	// Type of the condition, currently ('Ready').
+	Type InstanceConditionType `json:"type"`
+	// Status of the condition, one of ('True', 'False', 'Unknown').
+	Status ConditionStatus `json:"status"`
+	// Reason is a brief machine readable explanation for the condition's last
+	// transition.
+	Reason string `json:"reason"`
+	// Message is a human readable description of the details of the last
+	// transition, complementing reason.
+	Message string `json:"message"`
 }
 
 // InstanceConditionType represents a instance condition value
 type InstanceConditionType string
 
 const (
-	// InstanceConditionProvisioning represents that a given instance condition is in
-	// provisioning state
-	InstanceConditionProvisioning InstanceConditionType = "Provisioning"
```

Comment:
Can you just leave these in? I'm working on 248 and was using these, so no point in removing and then re-adding?